en/config/motors.md

```diff
@@ -0,0 +1,14 @@
+# Motor Checks
+
+After the airframe is setup and configured you should validate the motor assignment, spin direction and servo response:
+
+1. Remove all propellers, switch to Manual flight mode and arm the vehicle.
```

Comment:
It is specifically the Manual mode, since you then have direct throttle (and roll/pitch) control. It applies to both FW or MC (where Manual==Stabilized on MC).

Reply:
Would you still link to MC and FW Manual modes?

Reply:
Yes. I will fix and merge now.

Reply:
Thanks